docker-compose.yml

```
@@ -9,9 +9,9 @@ services:
     env_file:
       - ./.flaskenv
     environment:
-      - DATABASE_URI=sqlite:////db/map.db
+      - DATABASE_URI=${DATABASE_URI}
```

"dict"-style is preferred for env vars
```suggestion
       DATABASE_URI: ${DATABASE_URI}
```

---

You may not need to expose DATABASE_URI as an env var since you're not using a separate DB service yet...